base/compiler/ssair/inlining.jl

```diff
@@ -914,7 +914,6 @@ function analyze_method!(match::MethodMatch, argtypes::Vector{Any},
 end
 
 function InliningTodo(mi::MethodInstance, ir::IRCode, effects::Effects)
-    ir = copy(ir)
```

Comment:
Should this have been a separate PR? Seems like an odd thing to have in the commit without any description except "redundant"

Reply:
This copy was moved somewhere else in a conflicting PR while this PR was pending, so after a rebase there ended up being a duplicated copy call that wasn't in the original PR

Reply:
Oversight during the squash merge.

Reply:
ah, okay